WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -477,6 +481,7 @@ public boolean onPrepareOptionsMenu(Menu menu) {
                         } else {
                             saveMenuItem.setTitle(R.string.update_verb);
                         }
+                        mSaveWithoutLeaving = true;
```

Comment:
Just the "local draft" of a PUBLISHED or an UNKNOWN post. I'd assume we don't want to stay in the editor when the button says Publish, do we? The "default" case handles the DRAFT status.